#ifndef NETWORK_H
#define NETWORK_H
#include <QTcpSocket>
#include <QUdpSocket>
#include <QTcpServer>
#include <QNetworkInterface>
#include <QHostAddress>
#include <QMessageBox>
#include <QList>

#include "mainwindow.h"
#include "ui_mainwindow.h"
#include "form.h"
#include <QWidget>
#include <QThread>

#endif // NETWORK_H
